src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -0,0 +1,189 @@
+@import '~igz-controls/scss/shadows';
+@import '~igz-controls/scss/colors';
+
+.metrics {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  height: fit-content;
+
+  &__card-tmp {
+    z-index: 1;
+    top: 0;
+    position: sticky !important;
+    box-sizing: border-box;
+    border: 1px solid #00000033;
+    border-radius: 12px !important;
+    background: #fff;
+    padding-top: 12px !important;
+    width: 100%;
+    transition: 0.1s;
+    animation: ease-in-out;
+    max-height: 140px;
+  }
+
+  &__card {
+    border-radius: 12px;
+    border: 1px solid rgba(0, 0, 0, 0.2);
```

Comment:
Almost in all cases we use border variables in order not to duplicate it

Reply:
i will add all colors as a variables. (additional PR).